influence_simulator/__main__.py

```diff
@@ -1,21 +1,148 @@
+"""
+Script for simulating nodes influence in networks. 
+Simulations are run with the Independent Cascade Model
+"""
+
+import logging
 from argparse import ArgumentParser, Namespace, RawDescriptionHelpFormatter
+from datetime import datetime
+from pathlib import Path
+
+import numpy as np
+
+from influence_simulator.influence_simulator import InfluenceSimulator
+
+from .utils import log_memory, timer
+
+
+rng = np.random.default_rng()
+
+
+def setup_logging(log_level: str) -> None:
+    if log_level.lower() in ("none", "disable"):
+        log_level = logging.WARNING + 100
+    else:
+        log_level = log_level.upper()
+
+    logging.basicConfig(
+        level=log_level,
+        format="[%(levelname)s] %(message)s",
+    )
+
+
+def get_output_file(
```

Comment:
I would move those two functions to utils, but nothing necessary

Reply:
These functions are written specifically to the requirements of the scripts. They will be of no use outside (i.e. when imported by the outside package)